src/Microsoft.Health.Dicom.Functions/Migration/Models/MigrationFilesBatchArguments.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Functions.Migration.Models;
+
+/// <summary>
+///  Represents input to <see cref="MigrationFilesDurableFunction.MigrateFrameRangeFilesAsync"/>
+/// </summary>
+public sealed class MigrationFilesBatchArguments
```

Comment:
You could skip this class altogether too if you wanted such that argument type `WatermarkRange` directly

Reply:
True, initially I had extra property then removed and missed it can be simplified.